### DISCUSSION TITLE
nested read [Jul 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2704] [Discord Threads]
> 🙋‍♂️ **matte**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@848465010748817438> could you elaborate requirement with an example?    
####    
#### If you have a link established between table-a, table-b. And if you want to read a field belonging to table-b from table-a, you will have to add a lookup for that field in table-a.  
#  
> 🙋‍♂️ **matte**  
#### okey so it is not enoguht with only a relation?  
#  
> 🙋‍♂️ **matte**  
#### i was trying to use this https://docs.nocodb.com/developer-resources/rest-apis#nested-childrenhas-many. Where i thought it was possible to select what fields should be return from the child?  
#  
> 💁 **o1lab**  
#### http://localhost:9200/api/v1/db/data/noco/p_nok8ahxsgf6u7j/City/?nested[Country][fields]=LastUpdate,Country,CountryId  
#  
> 💁 **o1lab**  
#### City BelongsTo Country in above example  
#  
> 🙋‍♂️ **matte**  
#### Thank now i mad it work, for some reason it dont work in swagger  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/995259902824034315)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
